src/main/scala/diplomaticobjectmodel/logicaltree/LogicalTreeNode.scala

```diff
@@ -51,7 +51,8 @@ object LogicalModuleTree {
   }
 
   def resourceBindings(deviceOpt: () => Option[Device], maps: ArrayBuffer[ResourceBindingsMap]): ResourceBindings = deviceOpt() match {
-    case Some(device) => getResourceBindings(device, maps)
+    case Some(device) =>
+      getResourceBindings(device, maps)
```

Comment:
I think the previous style of having this on a single line is better and at least more consistent with the None case below.

Reply:
fixed